docs/deployment/thin-egress-app.md

```diff
@@ -8,16 +8,26 @@ hide_title: true
 
 The [Thin Egress App (TEA)](https://github.com/asfadmin/thin-egress-app) is an app running in Lambda that allows retrieving data from S3 using temporary links and provides URS integration.
 
-Note: If you are using the `cumulus` module, you will not need the information on this page, as Cumulus packages TEA within its `distribution` tf-module, which is pre-configured and included in our releases as a submodule of the `cumulus` module. However, if you are using individual Cumulus modules in your own Terraform configuration, or wish to configure TEA on your own, the information below is important to set up distribution.
+## Cumulus "Packaged" Deployment
 
-## Thin Egress App deployment
+If you are using the `cumulus` module or the core `distribution` module, you will not need to manually deploy TEA. Cumulus packages TEA within its `distribution` tf-module, which is pre-configured and included in our releases.
+
+By default, Core provides a one-to-one mapping where each protected and public bucket map to a path of the same name as the bucket.
+
+The `cumulus` and `distribution` modules provide a `bucket_map_key` input variable which allows you to specify the path to a custom [bucket map YAML](https://github.com/asfadmin/thin-egress-app#buckets-and-bucket-map).
```

Comment:
When we say we're providing the file, is this a local path or does the bucket map need to be in S3 and it's the S3 location?

Reply:
Good question - we should probably clarify we're talking about the system bucket here.  This is a pass through for https://github.com/asfadmin/thin-egress-app/blob/devel/terraform/variables.tf#L7

Our current config being passed from the distribution module is: 

  bucket_map_file                    = var.bucket_map_key == null ? aws_s3_bucket_object.bucket_map_yaml[0].key : var.bucket_map_key
  config_bucket                      = var.system_bucket

Reply:
fb4864cd5